source/expander.txt

```diff
@@ -0,0 +1,27 @@
+Expander
+========
+
+Expanders allow to dynamically hide or show information within a window or
+dialog. An expander can take a single widget that will be displayed when
+expanded.
+
+Expanders remain expanded until clicked again. When the state of an expander
+is changed, the "activate" signal is emitted.
+
+An expander can be programmatically expanded or retracted by passing `True` or
+`False` to :func:`Gtk.Expander.set_expanded`. Note that doing so causes the
+"activate" signal to be emitted.
+
+Adding More Widgets
+-------------------
+
+More widgets, such as :Gtk.Label: or :Gtk.Button:, can be added by fitting them
```

Comment:
I would rephrase it saying "More than one widget, such as a Label and Button, ..."